submissions/SergSenras/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,54 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Dweller {
+  constructor (species,name,gender,saying,friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
```

Comment:
There are more props that all dwellers share. One of the points of OOP is to generalize shared props to a base class.

Reply:
```suggestion
  class Dweller {
  constructor (species,name,gender,legs,saying,friends) {
    this.species = species;
    this.name = name;
    this.gender = gender;
    this.legs = legs;
    this.saying = saying;
    this.friends = friends;
  }
```
added one more property